timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -32,13 +32,17 @@ def __init__(self, index_name, sketch_id, timeline_id=None, **kwargs):
         self._rule = kwargs.get("rule")
         super().__init__(index_name, sketch_id, timeline_id=timeline_id)
 
-    def run_sigma_rule(self, query, rule_name, tag_list=None):
+    def run_sigma_rule(
+        self, query, rule_name, tag_list=None, status_good=True
+    ):
         """Runs a sigma rule and applies the appropriate tags.
 
         Args:
             query: OpenSearch search query for events to tag.
             rule_name: rule_name to apply to matching events.
             tag_list: a list of additional tags to be added to the event(s)
+            status_good: value from sigma_status.csv good / bad / exploratory
```

Comment:
This makes it sound as if we expect a string, but we're using a boolean here, so the comparison has to be made by the caller.